README.md

```diff
@@ -1,3 +1,32 @@
 # RHMAP Nagios for Docker
 
 Nagios server in a Docker container for RHMAP.
+
+# Setting up nagios service account
+
+In order to talk to the OpenShift and perform exec commands on pods, a service account will need to be created for use by nagios and given the proper permissions.
```

Comment:
I'm still missing some quick notes here, in the README, explaining why those specific permissions are required.